ui/lib/sync/addon/components/secrets/landing-cta.hbs

```diff
@@ -4,7 +4,7 @@
 ~}}
 
 <div class="box is-fullwidth is-sideless is-flex-between is-shadowless" data-test-cta-container>
-  {{#if (is-version "Enterprise")}}
+  {{#if this.version.isCommunity}}
```

Comment:
Was the `isCommunity` property added to the version service in another PR?

Reply:
Yes I believe it was updated as part of the reduce disclosure work.